frontend/src/components/VariantListPage/VariantListVariants.tsx

```diff
@@ -192,22 +192,17 @@ const VariantListVariants = (props: VariantListVariantsProps) => {
       </Text>
       <Box mb={4}>
         <Text>Tagged Groups</Text>
-        <Text>
-          This variant list contains {tagCounts.A} variant
-          {tagCounts.A !== 1 ? "s" : ""} tagged {taggedGroups.A.displayName}.
-        </Text>
-        <Text>
-          This variant list contains {tagCounts.B} variant
-          {tagCounts.B !== 1 ? "s" : ""} tagged {taggedGroups.B.displayName}.
-        </Text>
-        <Text>
-          This variant list contains {tagCounts.C} variant
-          {tagCounts.C !== 1 ? "s" : ""} tagged {taggedGroups.C.displayName}.
-        </Text>
-        <Text>
-          This variant list contains {tagCounts.D} variant
-          {tagCounts.D !== 1 ? "s" : ""} tagged {taggedGroups.D.displayName}.
-        </Text>
+        {Object.keys(taggedGroups).map((key) => {
+          const tagKey = key as TagKey;
+          const count = tagCounts[tagKey];
+          return (
+            <Text key={tagKey}>
+              This variant list contains {count} variant
+              {count !== 1 ? "s" : ""} tagged {taggedGroups[tagKey].displayName}
+              .
+            </Text>
+          );
```

Comment:
Nice refactor here. My only comment this time around is to move this text into the modal where users can edit the names of the tags, it seems like it might makes sense to have that modal be the "here's where you can interact with all things tags" place. 

That could even be a good place to have the future (not in this PR) functionality of "change the currently selected variants to be those in tag group N".